explorer/src/api/index.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  GATEWAYS_API,
+  MIXNODES_API,
+  VALIDATORS_API,
+  BLOCK_API,
+} from './constants';
+
+export class Api {
```

Comment:
I'll show you how to add typings to the methods so that the response objects have nice typings. Sadly the OpenAPI crate the Explorer API is using (https://crates.io/crates/rocket_okapi) doesn't have enough features to autogenerate them from yet.